### DISCUSSION TITLE
WebDAV映射网络驱动器，天翼云和百度云不能上传

### DISCUSSION THREAD

**Discussion by Nielson-cn** [AlistGo/alist#3240] [Q&A]
看了Q&A，RaiDrive因“空文件”造成不能百度云上传（不知道天翼云是否也是这个原因）。但我是用Windows“映射网络驱动器”和“添加网络位置”，似乎也不行（不管按默认的设置“302重定向”还是“本地代理”）。“阿里云”“本地存储”上传都OK。

请问：有什么软件可以通过webdav向百度云、天翼云上传文件吗？谢谢！

**Comment by 258599** [comment]
我的天翼云也是不能用RaiDrive上传，网页可以上传小文件，不能上传大文件，你是“映射网络驱动器”和“添加网络位置”填的什么地址？是http://127.0.0.1:5244/dav吗？我的提示无效文件夹，RaiDrive可以挂

**Reply by Nielson-cn** [reply]
> 我的天翼云也是不能用RaiDrive上传，网页可以上传小文件，不能上传大文件，你是“映射网络驱动器”和“添加网络位置”填的什么地址？是[http://127.0.0.1:5244/dav吗？我的提示无效文件夹，RaiDrive可以挂](http://127.0.0.1:5244/dav%E5%90%97%EF%BC%9F%E6%88%91%E7%9A%84%E6%8F%90%E7%A4%BA%E6%97%A0%E6%95%88%E6%96%87%E4%BB%B6%E5%A4%B9%EF%BC%8CRaiDrive%E5%8F%AF%E4%BB%A5%E6%8C%82)

地址对的。
可能是因为Win10默认不允许连http的WebDAV（只能https）。连http须要改注册表。网上有教程，比如这个：https://www.shuzhiduo.com/A/gVdnjX4EdW/

**Reply by 258599** [reply]
哈哈，磁盘映射成功了，谢谢！有点奇怪，你都能成功映射磁盘，干嘛还要用RaiDrive呢

**Reply by Nielson-cn** [reply]
> 哈哈，磁盘映射成功了，谢谢！有点奇怪，你都能成功映射磁盘，干嘛还要用RaiDrive呢

一来我最早是一直用RaidDrive的。当时用WebDAV并不频繁，用不着一开机就直接挂网盘。

二来Alist手册里写了RaidDrive为什么不能写入百度网盘（空文件问题），但没写天翼云、磁盘映射是否也如此。我写这个是说明“我已经查了手册等资料，但没找到答案”——这是提问者应有的礼数（参考《提问的智慧》）。

**Reply by 258599** [reply]
刚刚试了下，磁盘映射没什么用，只能操作小文件，大文件显示：错误0x800700DF:文件大小超出允许的限制，无法保存。可能是映射出来的文件系统是FAT。我也不太懂，不知道怎么映射成NTFS文件系统。还是谢谢你

**Reply by Nielson-cn** [reply]
> 刚刚试了下，磁盘映射没什么用，只能操作小文件，大文件显示：错误0x800700DF:文件大小超出允许的限制，无法保存。可能是映射出来的文件系统是FAT。我也不太懂，不知道怎么映射成NTFS文件系统。还是谢谢你

不客气~这也是我遇到的问题——挂阿里云OK，但挂百度网盘、天翼云就只能读不能写。

**Reply by liflux** [reply]
我放弃了，上传好像可以传大文件，下载不行。

**Reply by liflux** [reply]
我是用nautilus直接挂载的Webdav

**Comment by youtehub** [comment]
我一般都是用rclone上传文件的，rclone可以

**Reply by Nielson-cn** [reply]
> 我一般都是用rclone上传文件的，rclone可以

非常感谢。去看了一下，rclone是Alist的替代品，可惜似乎不支持百度云盘和天翼云盘……
